even_fib_nums.js

```diff
@@ -20,16 +39,62 @@ function _highestFibonacciNumber (maxFibValue){
   //define your base case, validate your input
 
 
-  //do your work here
+  var fibArray = [1,2];
+  function fibMaker(){
+    var fib1 = 1;
+    var fib2 = 2;
+    
+    var fibonacci = fib1 +fib2;
+    while(fibonacci<=maxFibValue){
+      fibonacci= fib1 + fib2;
+      fib1 = fib2;
+      fib2 = fibonacci;
+      fibArray.push(fibonacci)
+    }
+    return fibArray
+  }
+  fibMaker()
+  for(i=0;i<fibArray.length;i++){
```

Comment:
You could simplify this by making sure nothing is pushed into the array that's greater than `maxFibValue`.  Then instead of the a for loop, you can use `.pop` to get the last thing pushed into the array, which should be the higher fib number.